src/translation/french.c

```diff
@@ -65,18 +65,18 @@ static translation_string all_strings[] = {
     {TR_CONFIG_WOLVES_BLOCK, "Bloque la construction autour des loups"},
     {TR_CONFIG_DYNAMIC_GRANARIES, "Bloque les routes des greniers non connectées"},
     {TR_CONFIG_MORE_STOCKPILE, "Les maisons stockent plus de marchandises du marché"},
-    {TR_CONFIG_NO_BUYER_DISTRIBUTION, "Les vendeuses du marché ne distribuent pas les marchandises"},
+    {TR_CONFIG_NO_BUYER_DISTRIBUTION, "Les acheteuses du marché ne distribuent pas les marchandises"},
     {TR_CONFIG_IMMEDIATELY_DELETE_BUILDINGS, "Destruction immédiate des bâtiments"},
     {TR_CONFIG_GETTING_GRANARIES_GO_OFFROAD, "Les charretiers peuvent faire du hors route"},
     {TR_CONFIG_GRANARIES_GET_DOUBLE, "Double la capacité des charretiers sortant des greniers"},
     {TR_CONFIG_TOWER_SENTRIES_GO_OFFROAD, "Les sentinelles de la tour n'ont plus besoin de route"},
-    {TR_CONFIG_FARMS_DELIVER_CLOSE, "Fermes et Quais ne livrent plus qu'aux grenier à proximité"},
-    {TR_CONFIG_DELIVER_ONLY_TO_ACCEPTING_GRANARIES, "La nourriture n'est pas livrée depuis/vers les greniers"},
+    {TR_CONFIG_FARMS_DELIVER_CLOSE, "Fermes et quais ne livrent plus qu'aux grenier à proximité"},
+    {TR_CONFIG_DELIVER_ONLY_TO_ACCEPTING_GRANARIES, "Fermes et quais ne livrent plus aux greniers demandeurs"},
     {TR_CONFIG_ALL_HOUSES_MERGE, "Toutes les maisons fusionnent"},
     {TR_CONFIG_WINE_COUNTS_IF_OPEN_TRADE_ROUTE, "Ouvre des routes commerciales qui founissent différents types de vins"},
-    {TR_CONFIG_RANDOM_COLLAPSES_TAKE_MONEY, "Les mines coûtent de l'argent au lieu d'être détruites."},
+    {TR_CONFIG_RANDOM_COLLAPSES_TAKE_MONEY, "Les effondrements aléatoires coûtent de l'argent au lieu de détruire les bâtiments"},
```

Comment:
Have you confirmed this fits in the config window? If seems like a very large phrase to fit...